jerry-core/jerry.h

```diff
@@ -117,4 +122,7 @@ extern EXTERN_C void jerry_pop_ctx (void);
  * @}
  */
 
+#ifdef __cplusplus
+} /*  */
```

Comment:
Don't we want to add extern "C" to the comment here? (And all other similar places as wel.)